scripts/src/env/start.rs

```diff
@@ -26,15 +28,28 @@ pub async fn execute() -> anyhow::Result<Environment> {
 
     println!("✓");
 
+    print_progress!("Reading config file");
+
+    let path = std::env::current_dir()?.join(Path::new(config::FILE_NAME));
+    let config = match Config::from_file(&path) {
```

Comment:
You could destruct the `config` variable here directly into `(bitcoin_config, ethereum_config)` for increased readability! Index-access like `config.0` can be cryptic because it requires knowledge of what the tuple is composed of.